packages/store/addon/-private/system/core-store.ts

```diff
@@ -114,21 +114,6 @@ function deprecateTestRegistration(factoryType: 'serializer' | 'adapter', factor
   );
 }
 
-// Implementors Note:
```

Comment:
This seems unrelated? Is it just outdated / needing cleanup?

Reply:
jsut outdated